Jenkinsfile

```diff
@@ -3,7 +3,7 @@
 library("govuk")
 
 REPOSITORY = 'whitehall'
-DEFAULT_SCHEMA_BRANCH = 'deployed-to-production'
+DEFAULT_SCHEMA_BRANCH = 'main'
```

Comment:
Happy with all the changes here but it could be that this is better off in it's own PR, don't have massively strong feelings either way, and i guess actually the new system doesn't yet use release tags in the same way so just being in its own commit as is would be fine. Sorry for stream of consciousness, have approved